### PR TITLE
Remove auxiliary.py

### PR DESCRIPTION
**Pull request by MaxBlesch**
### What problem do you want to solve?

Right now we kept some functions in auxiliary.py, which are either not need in our new design or a one-liner. Therefore I will delete this module. Adressing partly #58 .

### Todo

- [x] Implement household and tax_unit aggregation directly in the functions.
- [x] Delte get_params and uprate.

### CODE REVIEW

**Comment by hmgaudecker** [comment]
Good idea to delete the entire module. Rather have five modules with names that tell us something than one container module with five unrelated functions.

File: gettsim/benefits/alg2.py [hunk 1]
```diff
@@ -33,7 +31,7 @@ def alg2(household, tb):
     )
     # Aggregate on HH
     for var in ["ar_alg2_ek", "alg2_grossek", "uhv"]:
-        household[var + "_hh"] = aggr(household, var, "all_hh")
+        household[var + "_hh"] = household[var].sum()
```
Comment (tobiasraabe) [inline gettsim/benefits/alg2.py]: Go for ``f"{var}_hh"`` which is more readable.

File: gettsim/benefits/kiz.py [hunk 2]
```diff
@@ -17,7 +15,7 @@ def kiz(household, tb):
         This is done by some fixed share which is updated on annual basis
         ('jährlicher Existenzminimumsbericht')
     """
-    household["uhv_tu"] = aggr(household, "uhv", "all_tu")
+    household["uhv_tu"] = household.groupby(["tu_id"])["uhv"].transform("sum")
```
Comment (tobiasraabe) [inline gettsim/benefits/kiz.py]: no need for squared brackets in groupby.

File: gettsim/tests/auxiliary_test_tax.py [hunk 3]
```diff
@@ -28,20 +28,19 @@ def load_test_data(year, file_name, columns, *pd_args, **pd_kwargs):
...
         for col in bool_cols:
             if col in df.columns:
                 df[col] = df[col].astype(bool)
-
-    df = df.loc[df["year"].eq(year), columns].copy()
-
-    return df
+    else:
+        raise ValueError("File does not exist.")
+    return df[df["year"] == year][columns]
```
Comment (tobiasraabe) [inline gettsim/tests/auxiliary_test_tax.py]: Do not assign on a return statement. ``df.loc[df.year.eq(year), columns].copy()`` avoids chained selection and probably prevents warnings like "do not alter view".

Comment (MaxBlesch) [reply gettsim/tests/auxiliary_test_tax.py]: Assigned before returning

File: gettsim/benefits/kiz.py (hunk 2 above)
Comment (MaxBlesch) [reply gettsim/benefits/kiz.py]: Fixed

File: gettsim/benefits/alg2.py (hunk 1 above)
Comment (MaxBlesch) [reply gettsim/benefits/alg2.py]: Converted to f-string

**Pull request merged**